testing_and_setup/compass/ocean/coastal/USDEQU120cr10rr2/build_mesh/config_culled_mesh.xml

```diff
@@ -14,8 +13,7 @@
 			<argument flag="--with_critical_passages"></argument>
 			<argument flag="--preserve_floodplain"></argument>
 		</step>
-		<step executable="python">
-			<argument flag="-m">jigsaw_to_MPAS.inject_bathymetry</argument>
+		<step executable="inject_bathymetry">
```

Comment:
Less is more-- nice!

Reply:
A happy side-effect rather than something I was explicitly doing to clean things up.